QUICK_START.md

```diff
@@ -6,14 +6,15 @@ This document provides quick access to several different ways to set up CORTX fo
     1. On a single node using Kubernetes: [LINK](doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst)
     1. On an AWS EC2 instance: [LINK](doc/integrations/AWS_EC2/README.md)
     1. Across a cluster: TODO 
-1. Run it on AWS using Kubernetes: [LINK](https://github.com/Seagate/cortx-k8s/blob/main/doc/cortx-aws-k8s-installation.md)
+1. Run CORTX on Kubernetes: [LINK](https://github.com/Seagate/cortx-k8s/blob/integration/README.md)
+    1. Run CORTX on Kubernetes using Amazon Web Services: [LINK](https://github.com/Seagate/cortx-k8s/blob/main/doc/cortx-aws-k8s-installation.md)
 1. Build entire CORTX into binaries: [LINK](./doc/community-build/docker/cortx-all/README.md)
-    1. Run it on a single node: [LINK](https://github.com/Seagate/cortx-k8s/blob/main/doc/cortx-aws-k8s-installation.md)        
+    1. Run CORTX on a single node: [LINK](https://github.com/Seagate/cortx-k8s/blob/main/doc/cortx-aws-k8s-installation.md)
```

Comment:
The link has 'aws' in it. If these instructions are specific to aws, then I think the text should reflect this. Do we even still need this link? Once it is built, then we just use the other instructions to run it. So maybe this build section should precede the run section.

Reply:
I didn't even catch the link target here. It's simply a duplicate of the 2.1 list item, since the document itself doesn't have an anchor link that specifies the "single node" deployment versus just a "regular deployment".